### DISCUSSION TITLE
Installation failed on linux docker

### DISCUSSION THREAD

**Discussion by kharabela** [fluvio-community/fluvio#3904] [Q&A]
[error.txt](https://github.com/infinyon/fluvio/files/14629801/error.txt) 
Attaching my log as text file.Not sure what went wrong here.Intel i5 with 8 gb RAM on linux kde plasma 6

**Comment by digikata** [comment]
The instructions have been updated a bit here: https://infinyon.com/docs/tutorials/docker-installation/  We added some commands to the docker compose to help with the configuration steps.

You can also find that same docker config in the fluvio repo: https://github.com/infinyon/fluvio/tree/master/examples/docker-compose
